### ISSUE TITLE
[docs] podman commit: document --format option

### ISSUE THREAD

**Issue by edsantiago** [projectatomic/libpod#765]
Basically, the `-m` option no longer works in `podman commit`:
```
# podman run --name mycontainer fedora date
Mon May 14 17:43:03 UTC 2018
# podman commit -m mmm mycontainer myimage
messages cannot be added to the OCIv1 image format.
```
Cause:  a824186ac9803ef5f7548df790988a4ebd2d9c07 on April 25. 

Workaround: add `--format docker`, or don't use `-m`. 

`podman-commit(1)` needs to be updated to (1) add a description of `--format` and its options, including the default `oci` value and preferably a link to some documentation of said format; and (2) include a note about `-m` being incompatible with this format.

**Comment by wking** [comment]
Fixed by #768.

**Comment by mheon** [comment]
Closing as such
